templates/dashboard/customer/list.html

```diff
@@ -52,8 +52,8 @@
                   <tr data-action-go="{% url "dashboard:customer-details" pk=customer.pk %}">
                     {% with customer.default_billing_address as default_address %}
                       <td>
-                        {% if default_address %}
-                          {{ default_address.first_name }} {{ default_address.last_name }}
+                        {% if customer.get_full_name != customer.email %}
```

Comment:
I'm not sure this check is necessary since `get_full_name` defaults to `email` if the full name wasn't assembled properly.

Reply:
If we don't check this then in some case email appears in 2 column in the customer list.